envi/codeflow.py

```diff
@@ -177,10 +177,11 @@ def addCodeFlow(self, va, arch=envi.ARCH_DEFAULT):
 
                 try:
                     # Handle a table branch by adding more branches...
+                    ptrfmt = ('<P', '>P')[self._mem.getEndian()]
```

Comment:
this is a flawed struct char "<P" since we're not sure what a pointer size is at this point.  however, that's for a different PR.  this is a change to the code flow.  we'll fix this part later.